### PR TITLE
docs: replace the example app using electron

### PR DESCRIPTION
**Pull request by iamaliyousefi**
#### Description of Change
[Atom editor](https://github.com/atom/atom) was used as an example of apps using Electron. Since [Atom editor has been archived](https://github.blog/2022-06-08-sunsetting-atom/) and is not maintained anymore, I replaced it with vscode as a more famous and live example of an app which uses Electron. 

#### Checklist

- [x ] PR description included and stakeholders cc'd

#### Release Notes

Notes: none

### CODE REVIEW

**Review by MarshallOfSound — DISMISSED** [review]
Atom was here originally because Electron was made specifically out of Atom.

We should probably just remove it from this list and not replace it, the links to `/apps` is enough

cc @electron/wg-ecosystem

**Comment by aysabzevar** [comment]
@MarshallOfSound  Thanks for your comment. I believe having a very well-known example of an app which used Electron is still beneficial.

**Comment by zcbenz** [comment]
This PR needs some opinions from @electron/wg-ecosystem .

**Review by erickzhao — APPROVED** [review]
docs: LGTM

**Review by jkleinsc — CHANGES_REQUESTED** [review]
Needs a small change to pass lint (no trailing spaces allowed).

File: README.md [hunk 1]
```diff
@@ -9,8 +9,8 @@ View these docs in other languages on our [Crowdin](https://crowdin.com/project/
 
 The Electron framework lets you write cross-platform desktop applications
 using JavaScript, HTML and CSS. It is based on [Node.js](https://nodejs.org/) and
-[Chromium](https://www.chromium.org) and is used by the [Atom
-editor](https://github.com/atom/atom) and many other [apps](https://electronjs.org/apps).
+[Chromium](https://www.chromium.org) and is used by the [Visual Studio
```
[the remark this answers is not in the document]
Comment (iamaliyousefi) [reply README.md]: Just removed the trailing space. Thanks again @jkleinsc  for the heads up.

**Review by iamaliyousefi — COMMENTED** [review]
Remove the trailing space.

**Review by jkleinsc — APPROVED** [review]

**Pull request merged**